internal/execution/target.go

```diff
@@ -0,0 +1,85 @@
+package execution
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"reflect"
+	"time"
+)
+
+type server struct {
+	server *httptest.Server
+	called bool
+}
+
+func (s *server) URL() string {
+	return s.server.URL
+}
+
+func (s *server) Close() {
+	s.server.Close()
+}
+
+func (s *server) Called() bool {
+	return s.called
+}
+
+func TestServerCall(
+	reqBody interface{},
+	sleep time.Duration,
+	statusCode int,
+	respBody interface{},
+) (string, func(), func() bool) {
+	server := &server{
+		called: false,
+	}
```

Comment:
server overwrites the server struct type, if no defaults use `new(server)`

Reply:
Removed the file, as the TestServerCall is only used in tests